cmd/backup-manager/app/import/import.go

```diff
@@ -94,6 +97,11 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
 		fmt.Sprintf("--d=%s", restorePath),
 		fmt.Sprintf("--tidb-port=%d", ro.Port),
 	}
+	if ro.TLSClient {
```

Comment:
Please help confirm if this is the correct setting as a TiDB Client as Lightning supports multiple certs.